hedera-node/src/main/java/com/hedera/services/state/submerkle/ExpirableTxnRecord.java

```diff
@@ -872,4 +892,9 @@ public void onlyExternalizeIfSuccessful() {
 			onlyExternalizedIfSuccessful = true;
 		}
 	}
+
+	// only used in unit tests
+	public void clearStakingRewardsPaid() {
```

Comment:
Code smell to expose methods only for tests. At minimum, it should be package private.